content/en/docs/concepts/configuration/secret.md

```diff
@@ -247,6 +247,7 @@ You can still [manually create](/docs/tasks/configure-pod-container/configure-se
 a service account token Secret; for example, if you need a token that never expires.
 However, using the [TokenRequest](/docs/reference/kubernetes-api/authentication-resources/token-request-v1/)
 subresource to obtain a token to access the API is recommended instead.
+The `kubectl create token` command can be used to obtain a token from the `TokenRequest` API.
```

Comment:
Got a link to how to use that command? Or to reference docs?

Reply:
`kubectl create token --help` has examples

Generated docs are updated in https://github.com/kubernetes/website/pull/33676#issuecomment-1126859819

Reply:
I mean for the doc. A link to instructions on how to use the command would be useful.

Reply:
updated with link to the command docs

Reply:
Thanks!